sdk/python/requirements/py3.10-requirements.txt

```diff
@@ -32,28 +32,28 @@ dask==2022.1.1
     # via feast (setup.py)
 dill==0.3.4
     # via feast (setup.py)
-fastapi==0.75.2
+fastapi==0.76.0
     # via feast (setup.py)
-fastavro==1.4.10
+fastavro==1.4.11
     # via
     #   feast (setup.py)
     #   pandavro
 fsspec==2022.3.0
     # via dask
-google-api-core==2.7.2
```

Comment:
should we be concerned that we went down a whole major version here? Is it even still supported?

Reply:
Hm this was specific to 3.10. Every other version has a major version of 1.